### ISSUE TITLE
propagating context to List of Nested

### ISSUE THREAD

**Issue by lauwers** [marshmallow-code/marshmallow#424]
fields.Nested(MySchema) picks up context from its parent schema. However, it doesn't seem that this is the case if I have a list of nested objects (i.e. fields.List(fields.Nested(MySchema)) )  Is there some magic to get context propagated all the way to MySchema in this case?

**Comment by sloria** [comment]
Have you tried doing `fields.Nested(MySchema(many=True)`?

**Comment by lauwers** [comment]
Apologies for not following-up sooner. It turns out that my original report was inaccurate, and context is indeed correctly propagated across fields.List(fields.Nested(MySchema)).

The real problem was that instead of using the “standard” fields.List(), I used my own—slightly modified—version (I have a “named entity list” requirement that is somewhat similar to what was asked in https://github.com/marshmallow-code/marshmallow/issues/120). In the process of adapting fields.List() for this purpose, it turns out I messed up the _add_to_schema() method (it failed to correctly set the “parent” on the nested schema). I corrected that problem and context is now correctly propagated.

thanks,

Chris
